vmdb/db/migrate/20150330133855_container_nodes_capacity.rb

```diff
@@ -0,0 +1,9 @@
+class ContainerNodesCapacity < ActiveRecord::Migration
+  def change
+    add_column :container_nodes, :capacity_num_cores, :int
+    add_column :container_nodes, :capacity_memory, :bigint
+    add_column :container_nodes, :infra_machine_id, :string
+    add_column :container_nodes, :machine_id, :string
+    add_column :container_nodes, :system_uuid, :string
```

Comment:
Let's try to match column names that are on Host.  Hardware specific values can be set via the hardware table.